src/Mailinglist/Client/MailinglistClient.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailinglist\Client;
+
+interface MailinglistClient
+{
+    public function subscribe(string $email, string $mailingListId): void;
```

Comment:
There is a `EmailAddress` value object, maybe good to use it to have a bit of validation?

Reply:
The old code didn't do any validation, and the frontend does not check any response, it does it own validation of the e-mail.
So I am happy to add it but without frontend work it would be quite pointless, and would never happen because there is already a JS check done before.

Reply:
Why is it pointless to add validation to an API that is public?